src/ostorlab/serve_app/common.py

```diff
@@ -274,3 +276,48 @@ def is_aab(file_content: bytes) -> bool:
             return False
     except zipfile.BadZipFile:
         return False
+
+
+def get_package_name(path: str) -> str:
+    """Get the package name of an android file.
+
+    Args:
+        path: Path to the android file.
+
+    Returns:
+        str: Package name of the android file.
+    """
+
+    apk_info = pyaxmlparser.APK(path)
+    return apk_info.get_package()
+
+
+def get_bundle_id(path: str) -> str:
+    """Get the bundle identifier of an iOS file.
+
+    Args:
+        path: Path to the iOS file.
+
+    Returns:
+        str: Bundle identifier of the iOS file.
+    """
+
+    with zipfile.ZipFile(path, "r") as z:
+        # Locate the Info.plist file within the IPA
+        infoplist = None
+        for filename in z.namelist():
+            dirs = filename.split("/")
+            # payload, Payload and payLoad the three formats can be found
+            if (
+                len(dirs) == 3
+                and dirs[0].lower().strip() == "payload"
+                and dirs[2].lower().strip() == "info.plist"
+            ):
+                content = z.open(filename).read()
+                infoplist = plistlib.loads(content)
+
+        if infoplist is None:
+            raise ValueError("Info.plist not found in the IPA")
+
+        # Extract and read the Info.plist file
+        return infoplist.get("CFBundleIdentifier", "")
```

Comment:
This should return None as we shoudl not create an IOSAsset without bundle id.

Reply:
Thats true for android and ios store. But for android file and ios file its not the case !